cosmoz-treenode-button-view.html

```diff
@@ -44,7 +47,7 @@
 		</style>
 		<div class="horizontal layout">
 			<paper-button class="flex" raised on-tap="openDialogTree">
-				<div class="pathToNode">[[ _getButtonLabel(nodesOnNodePath, buttonTextPlaceholder) ]]</div>
+				<div class="pathToNode" has-nodes$="[[ nodesOnNodePath.length ]]">[[ _getButtonLabel(nodesOnNodePath, buttonTextPlaceholder) ]]</div>
 			</paper-button>
 			<paper-icon-button icon="clear" on-tap="reset" hidden$="[[ !_enableReset(nodePath, noReset) ]]"></paper-icon-button>
```

Comment:
This is also an issue with `cosmoz-dialog` actually. The clear icon is visible until `nodePath` is set.
One solution could be to use a `visible` (i.e. opposite to hidden) attribute.  I mean the icon could have display: none by default, but become visible if `_enableReset` returns true